ci/build.yml

```diff
@@ -6,14 +6,13 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.8.3
+    tag: 1.15.2
```

Comment:
Not really a release issue, but if you're doing more work in this repo you might want to update the version to something newer than 1.15.2 (The latest 1.15.x version [on docker hub is 1.15.6](https://hub.docker.com/_/golang?tab=tags&page=1&ordering=last_updated&name=1.15))

Also it's definitely worth adding a `caches` declaration to the build and unit yamls as these significantly inprove build times on concourse as the go standard libraries and dependencies don't need to be dowloaded and built every time. There are examples in other repos eg. (https://github.com/ONSdigital/dp-image-api/blob/develop/ci/build.yml#L17)